src/ngraph/op/convolution.cpp

```diff
@@ -382,15 +411,13 @@ void op::v1::ConvolutionBackpropData::validate_and_infer_types()
                               "> Expected: ",
                               PadType::VALID);
 
-        if (data_pshape.is_static() && filters_pshape.is_static())
+        if (data_pshape.rank().is_static() && filters_pshape.is_static())
         {
-            const Shape& filters_shape = filters_pshape.to_shape();
-            const Shape& data_shape = data_pshape.to_shape();
+            vector<Dimension> data_shape{data_pshape}, filters_shape{filters_pshape}, output_shape;
 
-            Shape output_shape;
-            opset1::infer_conv_backprop_output_spatial_shape(
-                Shape{std::next(data_shape.begin(), 2), std::end(data_shape)},
-                Shape{std::next(filters_shape.begin(), 2), std::end(filters_shape)},
+            infer_conv_backprop_output_spatial_shape(
+                vector<Dimension>{std::next(data_shape.begin(), 2), std::end(data_shape)},
+                vector<Dimension>{std::next(filters_shape.begin(), 2), std::end(filters_shape)},
```

Comment:
@jane-intel Should these 2 vectors contain a range `< begin() + 2; end() )` or two values `begin() + 2` and `end()`? I think the way they are constructed up here will use a constructor that takes an initializer_list and thus both arguments to `infer_conv_backprop_output_spatial_shape` will contain 2 elements.